azurelinuxagent/ga/monitor.py

```diff
@@ -279,14 +281,10 @@ def daemon(self):
                 report_network_configuration_changes.log_network_configuration()
 
             while not self.stopped():
-                try:
-                    for op in periodic_operations:
-                        op.run()
-
-                except Exception as e:
-                    logger.error("An error occurred in the monitor thread main loop; will skip the current iteration.\n{0}", ustr(e))
-                finally:
```

Comment:
the finally is needed; we still want to sleep until the next operation should execute

Reply:
Is finally possible without try and my understanding is it never execute without try.

Reply:
you need try/finally (used to be try/except/finally)

Reply:
Addressed